fuel-core/Cargo.toml

```diff
@@ -28,12 +29,14 @@ env_logger = "0.9"
 fuel-asm = { version = "0.1", features = ["serde-types"] }
 fuel-storage = { version = "0.1" }
 fuel-tx = { version = "0.2", features = ["serde-types"] }
+fuel-txpool = { path = "../txpool", version = "0.2" }
 fuel-types = { version = "0.1", features = ["serde-types"] }
 fuel-vm = { version = "0.2", features = ["serde-types"] }
 futures = "0.3"
 graphql-parser = "0.3.0"
 hex = { version = "0.4", features = ["serde"] }
 hyper = "0.14"
+interfaces = { path = "../interfaces", package = "fuel-interfaces", version = "0.2" }
```

Comment:
Should use an exact version for local dependencies, to avoid headaches if we ever mess up semver. And makes find-and-replace easier.

Can you maybe rename this crate to...`fuel-interfaces`?

Reply:
It seem little bit strange, but i can see benefit, will be changed.